relay-server/src/actors/processor.rs

```diff
@@ -1036,6 +1036,7 @@ impl EnvelopeProcessorService {
             match relay_profiling::expand_profile(&item.payload()[..]) {
                 Ok(payloads) => new_profiles.extend(payloads),
                 Err(err) => {
+                    relay_log::debug!("invalid profile: {:#?}", err);
```

Comment:
are we able to see these logs in prod somewhere?

Reply:
No, that's local only.